heartbeat/sapdb.sh

```diff
@@ -44,11 +44,11 @@ background_check_saphostexec() {
 #                       misbehavior
 #
 cleanup_saphostexec() {
-  pkill -9 -f "$SAPHOSTEXEC"
-  pkill -9 -f "$SAPHOSTSRV"
-  oscolpid=`pgrep -f "$SAPHOSTOSCOL"`       # we check saposcol pid, because it
-                                            # might not run under control of
-					    # saphostexec
+  pkill -9 -f saphostexec
+  pkill -9 -f sapstartsrv
```

Comment:
Hi,
I got your point in general. But this change will cause new trouble.
The sapstartsrv executable may run multible times on a host. But not all instances must be related to this resource instance. E.g. they can also be related to a SAPInstance resource. Killing them all maybe causes damage to other resources.
Therefore it is necessary to identify the correct instance.
Do you have more details about the issue where it occured ?
IMO this can only happen, when the processes get startet manually on a console. Which should not be the case in a cluster environment.
Cheers
Alex